x/incentive/simulation/genesis.go

```diff
@@ -0,0 +1,150 @@
+package simulation
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	authexported "github.com/cosmos/cosmos-sdk/x/auth/exported"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/kava-labs/kava/x/incentive/types"
+)
+
+var (
+	CollateralDenoms    = [3]string{"bnb", "xrp", "btc"}
+	RewardDenom         = "ukava"
+	MaxTotalAssetReward = sdk.NewInt(1000)
```

Comment:
Increase this to 1000000000 - this is the total rewards per 24 or 48 hour period, so setting it this low would probably cause a lot of reward per second values to be 0. Related - let's set the minimum inflation in `kavadist` to 10%, so we can maximize the probability of having enough coins to payout rewards.

Reply:
https://github.com/Kava-Labs/kava/pull/439/commits/1b5f2889b5741ed550ff2f45a71f5a0e2d090b4e